examples/managed-identity/kubernetes-msi.json

```diff
@@ -42,8 +43,8 @@
       }
     },
     "servicePrincipalProfile": {
-      "servicePrincipalClientID": "",
-      "servicePrincipalClientSecret": ""
+      "servicePrincipalClientID": "msi",

```

Comment:
Actually, this is easier to do than I thought, and I agree with it in principle. I'll do that right now as well.